### PR TITLE
Use cpt to check for and install missing dependencies if any in Travis.

### PR DESCRIPTION
**Pull request by reikdas**

### CODE REVIEW

**Comment by reikdas** [comment]
The PR is ready.

File: .travis.yml [hunk 1]
```diff
@@ -52,6 +52,10 @@ matrix:
     - os: osx
       env: COMPILER=clang++ CCOMPILER=clang
       osx_image: xcode11
+      before_install:
+        - brew install swig;
```
Comment (vgvassilev) [inline .travis.yml]: Can you remind me again why do need this?

File: .travis.yml [hunk 2]
```diff
@@ -177,15 +189,17 @@ script:
...
+      --check-requirements \
       --current-dev=tar \
       --with-cling-url=https://github.com/$TRAVIS_REPO_SLUG \
       --with-clang-url=http://root.cern.ch/git/clang.git \
       --with-llvm-url=http://root.cern.ch/git/llvm.git \
       --with-cmake-flags="$CLING_BUILD_FLAGS" \
       --skip-cleanup
   else
-    tools/packaging/cpt.py \
+    echo yes | tools/packaging/cpt.py \
```
[the remark this answers is not in the document]
Comment (reikdas) [reply .travis.yml]: I somehow skipped over that flag. I replaced echo yes with the -y flag.

File: .travis.yml (hunk 1 above)
Comment (reikdas) [reply .travis.yml]: It was an oversight on my part :( I made the required change.

Comment (vgvassilev) [reply .travis.yml]: No worries.

**Comment by vgvassilev** [comment]
Could the last commit be squashed to a previous one?

**Comment by reikdas** [comment]
Done.

**Comment by vgvassilev** [comment]
Merged.

**Pull request closed without merging**